README.md

```diff
@@ -124,151 +86,43 @@ all:
-* `restarted` (`boolean`): flag indicates if instance should be
-  restarted or not (if this flag isn't specified, instance will be restarted if
-  it's needed to apply configuration changes);
-* `expelled` (`boolean`, default: `false`): boolean flag that indicates if instance must be expelled from topology;
-* `stateboard` (`boolean`, default: `false`): boolean flag that indicates
-   that the instance is a [stateboard](#stateboard-instance);
-* `instance_start_timeout` (`number`, default: `60`): time in seconds to wait for instance to be started;
-* `cartridge_wait_buckets_discovery` (`boolean`, default: `true`): boolean
-  flag that indicates if routers should wait for buckets discovery after vshard bootstrap;
-* `instance_discover_buckets_timeout` (`number`, default: `60`): time in seconds
-  to wait for instance to discover buckets;
+Then run the playbook with created inventory:
 
-Replicasets configuration:
+```bash
+ansible-playbook -i hosts.yml playbook.yml
+```
 
-* `replicaset_alias` (`string`): replicaset alias, will be displayed in Web UI;
-* `failover_priority` (`list-of-strings`): failover priority;
-* `roles` (`list-of-strings`, required if `replicaset_alias` specified): roles to be enabled on the replicaset;
-* `all_rw` (`boolean`): indicates that that all servers in the replicaset should be read-write;
-* `weight` (`number`): vshard replicaset weight (matters only if `vshard-storage` role is enabled);
+Now, visit  http://localhost:8081
 
-Cluster configuration
+![image](/doc/images/quck-start.png)
 
-* `cartridge_bootstrap_vshard` (`boolean`, default: `false`): boolean
-  flag that indicates if vshard should be bootstrapped;
-* `cartridge_app_config` (`dict`): application config sections to patch;
-* `cartridge_auth`: (`dict`): [authorization configuration](#cartridge-authorization);
-* `cartridge_failover_params` (`dict`): [failover](#failover) parameters;
-* [DEPRECATED] `cartridge_failover` (`boolean`): boolean flag that
-  indicates if eventual failover should be enabled or disabled;
-
-### Role tags
-
-This role tasks have special tags that allows to perform only specified actions.
-Tasks are running in this order:
-
-* `cartridge-instances` - install package, update instances config and restart instances;
-* `cartridge-replicasets` - probe instances, configure replicasets, expel instances;
-* `cartridge-config` - configuring a cluster contains this tasks:
-  * configure authorization (if `cartridge_auth` is defined);
-  * patch application clusterwide config (if `cartridge_app_config` is defined);
-  * bootstrap Vshard (if `cartridge_bootstrap_vshard` is defined);
-  * manage cartridge failover (if `cartridge_failover_params` is defined).
-
-**Note**, that `cartridge-config` tasks would be skipped if no one of `cartridge_auth`, `cartridge_app_config`, `cartridge_bootstrap_vshard` and `cartridge_failover_params` variables is specified.
-
-### Using scenario
+## Using scenario
```

Comment:
All sections of the config have been moved, except quick start guide. Why did this section stay here? It's not for quick start, it's for complex usage